fw/tls.c

```diff
@@ -754,20 +761,20 @@ tfw_tls_conn_send(TfwConn *c, TfwMsg *msg)
 {
 	int r;
 	TlsCtx *tls = tfw_tls_context(c);
-	TlsIOCtx *io = &tls->io_out;
 
 	/*
 	 * Only HTTP messages go this way, other (service) TLS records are sent
 	 * by tfw_tls_send().
 	 */
-	io->msgtype = TTLS_MSG_APPLICATION_DATA;
 	T_DBG("TLS %lu bytes (%u bytes, type %#x)"
 	      " are to be sent on conn=%pK/sk_write_xmit=%pK ready=%d\n",
-	      msg->len, io->msglen + TLS_HEADER_SIZE, io->msgtype, c,
-	      c->sk->sk_write_xmit, ttls_xfrm_ready(tls));
+	      msg->len, io->msglen + TLS_HEADER_SIZE, TTLS_MSG_APPLICATION_DATA,
```

Comment:
This is debug messaging, but I believe there is no sense to log the constant - let's just remove it

Reply:
fixed